com.microsoft.mrtk.uxcore/Slider/Slider.cs

```diff
@@ -92,11 +93,55 @@ public Collider TrackCollider
             set => trackCollider = value;
         }
 
-        [Range(minVal, maxVal)]
+        [SerializeField]
+        [Tooltip("The minimum value that the slider can take on")]
+        private float minValue = 0.0f;
+
+        /// <summary>
+        /// The minimum value that the slider can take on
+        /// </summary>
+        public float MinValue
+        {
+            get
+            {
+                return minValue;
+            }
+            set
+            {
+                minValue = Mathf.Min(value, maxValue);
+            }
+        }
+
+        [SerializeField]
+        [Tooltip("The maximum value that the slider can take on")]
+        private float maxValue = 1.0f;
+
+        /// <summary>
+        /// The maximum value that the slider can take on
+        /// </summary>
+        public float MaxValue
+        {
+            get
+            {
+                return maxValue;
+            }
+            set
+            {
+                maxValue = Mathf.Max(minValue, value);
+            }
+        }
+
+        [VariableRange("minValue", "maxValue")]
         [SerializeField]
         private float sliderValue = 0.5f;
 
-        public float SliderValue
+        [Obsolete("Use Value instead")]
```

Comment:
Do we want to rename the backing field too?

Reply:
Yeah, I'll mark this PR as a breaking change as well